CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The most recent changes are on top, in each type of changes category.
 
 ### Added
 
+- Password change front [#158] (https://github.com/cyfronet-fid/sat4envi/issues/158)
```

Comment:
Don't modify already released version. Add the entry to the Unreleased section above.

Reply:
Well, it's not resolved in the repo though

Reply:
I just pushed changes